### PR TITLE
add skin terms

### LINKED ISSUE #2996: NTRs: skin of lobe of tail, skin of shin, skin of scapula region, skin of gonad
**Preferred term label:** skin of lobe of tail
**narrow synonym**: fluke skin PMID:21959845
**Definition** A zone of skin that is part of the lobe tail.
https://orcid.org/0000-0002-0819-0473
**Parent term**
zone of skin, UBERON:0000014
'part of' some ‘lobe of tail’ UBERON_0013131
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

**Preferred term label:** skin of shin
**Definition** A zone of skin on the front hindlimb zeugopod.
https://orcid.org/0000-0002-0819-0473
**Parent term**
zone of skin, UBERON:0000014
'part of' some 'front hindlimb zeugopod' UBERON_8480049
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

**Preferred term label:** skin of scapula region
**Definition** A zone of skin that is above the scapula in the upper back region.
https://orcid.org/0000-0002-0819-0473
**Parent term**
zone of skin, UBERON:0000014
'part of' some 'dorsal thoracic segment of trunk’
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

**Preferred term label:** skin of gonad
**Definition** A zone of skin part of some gonad.
https://orcid.org/0000-0002-0819-0473
**Parent term**
zone of skin, UBERON:0000014
'part of' some 'gonad’
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2996

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0013131 ! lobe of tail
+property_value: dcterms-date "2023-08-29T09:37:23Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480072
+name: skin of shin
+def: "A zone of skin on the front hindlimb zeugopod." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:224224]: Consider adding logical definition 'zone of skin' and ('part of' some 'front hindlimb zeugopod')

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: located_in UBERON:0002371 ! bone marrow
 property_value: dcterms-date "2023-06-26T13:48:30Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480071
+name: skin of lobe of tail
+def: "A zone of skin that is part of the lobe tail." [https://orcid.org/0000-0002-0819-0473]
+synonym: "fluke skin" NARROW [PMID:21959845]
+is_a: UBERON:0000014 ! zone of skin
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:224215]: Consider adding logical definition 'zone of skin' and ('part of' some 'lobe of tail')

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0008231 ! dorsal thoracic segment of trunk
+property_value: dcterms-date "2023-08-29T09:53:33Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480074
+name: skin of gonad
+def: "A zone of skin part of some gonad." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:224242]: Consider adding logical definition 'zone of skin' and ('part of' some gonad)

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+relationship: part_of UBERON:0003823 ! hindlimb zeugopod
+property_value: dcterms-date "2023-08-29T09:47:32Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480073
+name: skin of scapula region
+def: "A zone of skin that is above the scapula in the upper back region." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0008231 ! dorsal thoracic segment of trunk
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:224235]: consider using [upper back skin](http://purl.obolibrary.org/obo/UBERON_0009015) (it is part_of dorsal thoracic segment of trunk)

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please see inline comments.
I have done a cursory search of these terms but I could not find references, I suppose that is why you use your ORCID as dbxrefs.

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
...
+relationship: part_of UBERON:0013131 ! lobe of tail
+property_value: dcterms-date "2023-08-29T09:37:23Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480072
+name: skin of shin
+def: "A zone of skin on the front hindlimb zeugopod." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0003823 ! hindlimb zeugopod
```
[the remark this answers is not in the document]
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224226]: @aleixpuigb thanks for the spotting this. I made the change.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224224]: @aleixpuigb Agree with that. We need this term to describe the cell line SCCT10 (CVCL_VJ56) originated in PMID:24662767 (Supp Data Table 4  "left shin").

File: src/ontology/uberon-edit.obo (hunk 3 above)
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224242]: @aleixpuigb OK. We need this term to describe the cell line GM01722 (CVCL_JC93) originated from a patient whose sex is ambiguous. "She is female but "X,Y".

**Comment by paulacalipho** [comment]
@aleixpuigb Thanks for reviewing these terms. I did not find definitions for these terms in the literature. I commented in each case for which cell lines we need these terms and where they are used in the literature. I agree to add the logical definitions, but I do not know how to do it in Protege. I will be happy if you could show me how. For the moment I made all the other changes.

File: src/ontology/uberon-edit.obo (hunk 4 above)
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224235]: @aleixpuigb does this term require only is_a 'upper back skin' without any additional relationship: part_of ?

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224235]: I would say that the skin of the scapula region is part_of some upper back skin.

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224235]: @aleixpuigb OK. Should I also add here a logical definition 'zone of skin' and ('part of' some 'upper back skin')?

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224235]: OK

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224235]: > @aleixpuigb does this term require only is_a 'upper back skin' without any additional relationship: part_of ?

Sorry @paulacalipho, you were right in this comment. Looking at other examples, a scapular skin is_a 'upper back skin'.

**Pull request merged**